src/content/configuration/output.md

```diff
@@ -1088,4 +1088,36 @@ module.exports = {
     compareBeforeEmit: false
   }
 };
-```
\ No newline at end of file
+```
+
+## `output.iife`
+
+`boolean = true`
+
+Tells webpack to add iife wrapper around emitted code
+
+```javascript
+module.exports = {
+  //...
+  output: {
+    iife: true
+  }
+};
+```
+
+## `output.module`
+
+`boolean = true`
+
+Allow outputting javascript files as module type. It set  `output.iife` to `false`, `output.libraryTarget` to `"module"`, `output.jsonpScriptType` to `"module"` and `terserOptions.module` = `true`
```

Comment:
```suggestion
Allow outputting javascript files as module type. It sets  `output.iife` to `false`, `output.libraryTarget` to `"module"`, `output.jsonpScriptType` to `"module"` and `terserOptions.module` = `true`
```

Reply:
I wasn't allowed to apply this suggestion after applying suggestion from @montogeek on the same line. I have tagged you into the commit instead.